modules/post/windows/gather/enum_av_excluded.rb

```diff
@@ -0,0 +1,133 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require 'rex'
+
+class Metasploit3 < Msf::Post
+
+  include Msf::Post::Windows::Registry
+
+  def initialize(info={})
+    super( update_info( info,
+        'Name'          => 'Windows Antivirus Excluded Locations Enumeration',
+        'Description'   => %q{ This module will enumerate all excluded directories within supported AV products },
+        'License'       => MSF_LICENSE,
+        'Author'        => [ 'Andrew Smith'],
+        'Platform'      => [ 'win' ],
+        'SessionTypes'  => [ 'meterpreter' ]
+      ))
+
+  end
+
+  def enum_mssec
+    if registry_enumkeys("HKLM\\SOFTWARE\\Microsoft").include?("Microsoft Antimalware")
+        print_status "MS Security Essentials Identified"
+        return true
+    else
+        return false
+    end
+  rescue
+    return false
+  end
+  def enum_defender
+    if registry_enumkeys("HKLM\\SOFTWARE\\Microsoft").include?("Windows Defender")
+        print_status "Windows Defender Identified"
```

Comment:
For every `print_status` you are using when the product is found, it would be helpful to print the key where this was found.  So you'd have to change this to something like:

```
if (key = registry_enumkeys(somepath).find?(something))
  print_status("Found blah at #{key}")
  return true
else
  return false
end
```